packages/contracts/test/SphinxUtils.t.sol

```diff
@@ -14,6 +15,154 @@ import {
 contract SphinxUtils_Test is Test, SphinxUtils {
     function setUp() public {}
 
+    function test_checkFork_passes() external {
+        assertEq(checkFork(), true);
+    }
+
+    function test_sphinxCheckAccesses_fails_did_not_use_default_factory() external {
+        // Deploy the contract so we can get the correct deployed code
+        SphinxForkCheck forkCheck = new SphinxForkCheck();
+        address expectedAddress = vm.computeCreate2Address(
+            0,
+            keccak256(type(SphinxForkCheck).creationCode),
+            DETERMINISTIC_DEPLOYMENT_PROXY
+        );
+
+        Vm.AccountAccess[] memory accountAccesses = new Vm.AccountAccess[](2);
+        accountAccesses[0] = VmSafe.AccountAccess({
```

Comment:
In the future, it'd be helpful if you could document what's supposed to be different about test cases like this that involve boilerplate. Takes some effort for me to figure out that e.g. `accountAccesses[0].account` is the variable that we're testing here